dist/gemwallet-api.js

```diff
@@ -1 +1 @@
ction(e){return r(void 0,void 0,void 0,(function(){var t,n,r,c,l,s;return o(this,(function(o){switch(o.label){case 0:t={type:"reject",result:void 0},o.label=1;case 1:return o.trys.push([1,3,,4]),n={app:i.GEM_WALLET,type:"REQUEST_SUBMIT_TRANSACTION/V3",payload:e},[4,(0,u.sendMessageToContentScript)(n)];case 2:if(r=o.sent(),c=r.result,l=r.error,s=l?(0,a.deserializeError)(l):void 0)throw s;return c&&(t.type="response",t.result=c),[3,4];case 3:throw o.sent();case 4:return[2,t]}}))}))}},285:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0})},406:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0}),t.GEM_WALLET=void 0,t.GEM_WALLET="gem-wallet"},217:function(e,t,n){var r=this&&this.__createBinding||(Object.create?function(e,t,n,r){void 0===r&&(r=n),Object.defineProperty(e,r,{enumerable:!0,get:function(){return t[n]}})}:function(e,t,n,r){void 0===r&&(r=n),e[r]=t[n]}),o=this&&this.__exportStar||function(e,t){for(var n in e)"default"===n||Object.prototype.hasOwnProperty.call(t,n)||r(t,e,n)};Object.defineProperty(t,"__esModule",{value:!0}),o(n(285),t),o(n(311),t),o(n(569),t),o(n(290),t),o(n(406),t),o(n(726),t),o(n(780),t),o(n(875),t)},311:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0}),t.MSG_INTERNAL_RECEIVE_SIGN_OUT=t.MSG_INTERNAL_RECEIVE_PASSWORD=t.MSG_INTERNAL_REQUEST_PASSWORD=void 0,t.MSG_INTERNAL_REQUEST_PASSWORD="INTERNAL_REQUEST_PASSWORD",t.MSG_INTERNAL_RECEIVE_PASSWORD="INTERNAL_RECEIVE_PASSWORD",t.MSG_INTERNAL_RECEIVE_SIGN_OUT="INTERNAL_RECEIVE_SIGN_OUT"},569:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0})},290:(e,t)=>{var n,r,o;Object.defineProperty(t,"__esModule",{value:!0}),t.NETWORK=t.NetworkServer=t.Network=void 0,function(e){e.MAINNET="Mainnet",e.TESTNET="Testnet",e.DEVNET="Devnet",e.AMM_DEVNET="AMM-Devnet",e.CUSTOM="Custom"}(r=t.Network||(t.Network={})),function(e){e.MAINNET="wss://xrplcluster.com",e.TESTNET="wss://s.altnet.rippletest.net:51233",e.DEVNET="wss://s.devnet.rippletest.net:51233",e.AMM_DEVNET="wss://amm.devnet.rippletest.net:51233"}(o=t.NetworkServer||(t.NetworkServer={})),t.NETWORK=((n={})[r.MAINNET]={name:r.MAINNET,server:o.MAINNET,description:"Main network using the production version of the XRP Ledger."},n[r.TESTNET]={name:r.TESTNET,server:o.TESTNET,description:"Acts as a testing network, without impacting production users and risking real money."},n[r.DEVNET]={name:r.DEVNET,server:o.DEVNET,description:"A preview of upcoming features, where unstable changes are tested out."},n[r.AMM_DEVNET]={name:r.AMM_DEVNET,server:o.AMM_DEVNET,description:"XLS-30d Automated Market Makers preview network."},n[r.CUSTOM]={name:r.CUSTOM,server:"",description:"Custom network"},n)},726:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0})},780:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0})},875:(e,t)=>{Object.defineProperty(t,"__esModule",{value:!0})}},t={};return function n(r){var o=t[r];if(void 0!==o)return o.exports;var i=t[r]={exports:{}};return e[r].call(i.exports,i,i.exports,n),i.exports}(607)})()));
```

Comment:
Would it be possible to not commit this file at all? Since the API is untouched, it's safer not to modify the dist file

Reply:
I agree, no changes were made to this file. Not committing shouldn't cause any issues.